src/rgw/rgw_iam_policy.h

```diff
@@ -145,7 +145,8 @@ static constexpr std::uint64_t snsGetTopicAttributes = stsAll + 1;
 static constexpr std::uint64_t snsDeleteTopic = stsAll + 2;
 static constexpr std::uint64_t snsPublish = stsAll + 3;
 static constexpr std::uint64_t snsSetTopicAttributes = stsAll + 4;
-static constexpr std::uint64_t snsAll = stsAll + 5;
+static constexpr std::uint64_t snsCreateTopic = stsAll + 5;
```

Comment:
please add this to the [actpairs[] array](https://github.com/ceph/ceph/blob/f4758e5/src/rgw/rgw_iam_policy.cc#L64-L65) and to the switch statement in [action_bit_string()](https://github.com/ceph/ceph/blob/f4758e5/src/rgw/rgw_iam_policy.cc#L1203-L1206)